### ISSUE TITLE
enable transpilation of selected region only

### ISSUE THREAD

**Issue by justin-hackin** [edasque/sublime-lebab#2]
could be useful when I paste some ES5 code into my ES6 project, or to prevent certain code from being transpiled in lieu of #1

**Comment by edasque** [comment]
I agree, don't have much time to address but feel free to submit a PR.
